.github/workflows/hugo.yml

```diff
@@ -4,7 +4,7 @@ name: Deploy Hugo site to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["main"]
+    branches: ["master"]
```

Comment:
/it might make sense to switch the repo to `main`, WDYT?

Reply:
Yeah, that seems to be a good move. I'm doing the switch.